lib/boot.js

```diff
@@ -2,10 +2,6 @@ import service from './service';
 import config from './config';
 import ZKClient from './upstream/zookeeper';
```

Comment:
What zookeeper client are you using?  Have you talked to Frontend about their problems with Odysseus?